ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -0,0 +1,590 @@
+            </div>
+          </div>
+          <div style={{display: 'inline-block'}}>
+            <div style={{display: 'flex'}}>
+              <WorkspaceCategory item={ResearchPurposeItems.controlSet}
+                  value={this.state.workspace.researchPurpose.controlSet}
+                  onChange={v => this.updateWorkspaceCategory('controlSet', v)}/>
+              <WorkspaceCategory item={ResearchPurposeItems.ancestry}
+                  value={this.state.workspace.researchPurpose.ancestry}
+                  onChange={v => this.updateWorkspaceCategory('ancestry', v)}/>
+            </div>
+          </div>
+          <div style={{display: 'inline-block'}}>
+            <div style={{display: 'flex'}}>
+              <WorkspaceCategory item={ResearchPurposeItems.population}
+                  value={this.state.workspace.researchPurpose.population}
+                  onChange={v => this.updateWorkspaceCategory('population', v)}/>
+              <WorkspaceCategory item={ResearchPurposeItems.commercialPurpose}
+                  value={this.state.workspace.researchPurpose.commercialPurpose}
+                  onChange={v => this.updateWorkspaceCategory('commercialPurpose', v)}/>
+            </div>
+          </div>
+          <WorkspaceUnderservedPopulation
+              value={this.state.workspace.researchPurpose.underservedPopulationDetails}
+              onChange={v => this.updateUnderserverPopulation(v)}>
+          </WorkspaceUnderservedPopulation>
+        </WorkspaceEditSection>
+        <WorkspaceEditSection header='Request a review of your research purpose'
+                              tooltip={toolTipText.reviewRequest}>
+          <div style={{display: 'flex', flexDirection: 'row',
+            paddingBottom: '14.4px', paddingTop: '0.3rem'}}>
+            <input style={{height: '.66667rem', marginRight: '.31667rem', marginTop: '0.3rem'}}
+                   type='checkbox'
+                   onChange={v =>
+                   this.setState(fp.set(['workspace', 'researchPurpose', 'reviewRequested' ],
+                     v.target.checked))}
+                   checked={this.state.workspace.researchPurpose.reviewRequested}/>
+            <label style={styles.text}>
+              I am concerned about potential
+              <a onClick= {() => this.openStigmatization()}> stigmatization </a>
+            of research participants. I would like the All of Us Resource Access Board (RAB) to
+              review my Research Purpose.
+              (This will not prevent you from creating a workspace and proceeding.)
+            </label>
+          </div>
+        </WorkspaceEditSection>
+        <div>
+          <div style={{display: 'flex', flexDirection: 'row', marginTop: '1rem',
+            marginBottom: '1rem'}}>
+            <Button type='secondary' style={{marginRight: '1rem'}}>
+              Cancel
+            </Button>
+            <TooltipTrigger content={[<ul>Missing Required Fields:
```

Comment:
Optional: For some reason, this tooltip appears super far from the button. This appears to exist on test as well, so no big deal, but if you wanted to fix it, wanted to bring it up.